jaxopt/_src/gauss_newton.py

```diff
@@ -69,6 +68,7 @@ class GaussNewton(base.IterativeSolver):
   maxiter: int = 30
   tol: float = 1e-5
   verbose: bool = False
+  implicit_diff: bool = False
```

Comment:
We tend to use `True` as the default in other solvers.

Reply:
Thanks, applied.